src/Service/RequestService.php

```diff
@@ -568,7 +571,7 @@ public function proxyHandler(array $data, array $configuration, ?Source $proxy =
         // Make sure we set object to null in the session, for detecting the correct AuditTrails to create. Also used for DateRead to work correctly!
         $this->session->set('object', null);
 
-        if (isset($data['path']['{route}']) === true) {
+        if (isset($data['path']['{route}']) === true && empty($data['path']['{route}']) === false) {
```

Comment:
isn't `isset($data['path']['{route}']) === true` and `empty($data['path']['{route}']) === false` the same thing?